ldms/src/sampler/lustre_client/Plugin_lustre_client.man

```diff
@@ -33,9 +33,14 @@ name=<plugin_name>
 .br
 This MUST be lustre_client.
 .TP
-job_set=<metric set name>
+job_set=<job metric set name>
 .br
 The name of the metric set that contains the job id information (default=job_id)
+.TP
+instance=<metric set prefix>
+.br
+The prefix used to construct the set instance names. If not given, this defaults to
+the result of gethostname(). The set instance names will be $instance/$llite_name.
```

Comment:
The default is hostname/llite_name, not just the hostname.

Reply:
You seem to be confusing (default) value of the prefix with the constructed set name. If you think it would be more clear, this could be reworded as:
```
The set prefix used to construct the set names. If not given, this defaults to the result of gethostname(). The set names will be $prefix/$llite_name.
```